openerp/addons/base/res/res_company.py

```diff
@@ -229,13 +229,14 @@ def _company_default_get(self, cr, uid, object=False, field=False, context=None)
         if not context:
             context = {}
         proxy = self.pool.get('multi_company.default')
+        user = self.pool.get('res.users').browse(cr, SUPERUSER_ID, uid, context=context)
```

Comment:
Well you have two approaches to consider the `_*()` methods. Internal use (low level methods we should not use) or private methods (should not be called from rpc). We tend to mix both but here I believe it's the second one. Shall we create `__*()`methods? :-)